src/jobs/models/database/sequelize/sequelizeConnector.js

```diff
@@ -182,9 +182,9 @@ async function initSchemas() {
         }
     });
 
-    job.webhook = job.hasMany(webhook);
+    // job.webhook = job.hasMany(webhook);
     job.email = job.hasMany(email);
     await job.sync();
-    await webhook.sync();
+    // await webhook.sync();
```

Comment:
delete

Reply:
done, added the proper jobs-webhooks N:M association